packages/react/src/components/Modal/Modal.js

```diff
@@ -294,12 +294,16 @@ const Modal = React.forwardRef(function Modal(
   return (
     <div
       {...rest}
-      onKeyDown={handleKeyDown}
-      onMouseDown={handleMousedown}
-      onBlur={handleBlur}
       className={modalClasses}
+      ref={ref}
       role="presentation"
-      ref={ref}>
+      onKeyDown={handleKeyDown}>
+      <div
+        className={`${prefix}--modal-background`}
+        onMouseDown={handleMousedown}
+        onBlur={handleBlur}
+        aria-hidden
+      />
```

Comment:
Framework implementation authors (web components, svelte, angular, etc.) will need to make this change as well, correct? Without it, after taking this update their modal's are probably going to break, right?

Reply:
@davidmenendez Could you outline why this new separate element is needed?

Reply:
i took inspiration from some other modal implementations i had seen that used this pattern, but that's a valid point about breaking changes. i'll revert it as i think i can make it work without it